mopro-ffi/src/app_config/mod.rs

```diff
@@ -55,3 +56,28 @@ pub fn install_arch(arch: String) {
         .wait()
         .unwrap_or_else(|_| panic!("Failed to install target architecture {}", arch));
 }
+
```

Comment:
If we don't specify the lib name, the output lib will be named as the package name. The only difference is '-' is replace with '_'. So, here I just fetch the package name and replace '-'. In addition, still keep the flexibility to let users can specify the lib name by setting `name` under `[lib]` section in the toml file.